conda-forge.yml

```diff
@@ -1,13 +1,13 @@
 build_platform:
+  linux_aarch64: linux_64
+  linux_ppc64le: linux_64
   osx_arm64: osx_64
 conda_forge_output_validation: true
+github:
+  branch_name: main
+  tooling_branch_name: main
 provider:
   linux_aarch64: default
   linux_ppc64le: default
   win: azure
 test_on_native_only: true
-github:
-  branch_name: main
-  tooling_branch_name: main
-conda_build:
-  pkg_format: '2'
```

Comment:
The last merge removed this. Can you please add it back?

Reply:
Thanks, good catch!